spec/inquisition/factory_bot/runner_spec.rb

```diff
@@ -0,0 +1,28 @@
+RSpec.describe Inquisition::FactoryBot::Runner do
+  let(:runner) { described_class.new }
+  let(:factory_name) { 'foo' }
+  let(:message) { 'bar' }
+  let(:key_error) { instance_double(KeyError, message: message) }
+  let(:factory) { instance_double(FactoryBot::Factory, name: factory_name) }
+  let(:error) { FactoryBot::Linter::FactoryError.new(key_error, factory) }
+  let(:factory_hash_error) { { factory => [error] } }
+  let(:low_severity_issue) { { severity: :low, path: factory_name, message: "#{factory_name} - #{message}" } }
+
+  describe '#call' do
+    before do
+      allow(runner).to receive(:load_environment).and_return(true)
+      linter = instance_double(FactoryBot::Linter)
```

Comment:
I think would be better to move `instance_double(FactoryBot::Linter)` in `let()`